pkg/config/config.go

```diff
@@ -130,21 +132,48 @@ func (c *ConfigInfo) writeToFile() error {
 }
 
 // SetConfiguration modifies Odo configurations in the config file
-// as of now only being used for updatenotification
-func (c *ConfigInfo) SetConfiguration(parameter string, value bool) error {
+// as of now being used for timeout, updatenotification
+func (c *ConfigInfo) SetConfiguration(parameter string, value interface{}) error {
 	switch parameter {
+
+	case "timeout":
+		typedval, ok := value.(int)
+		if typedval == 0 {
+			fmt.Println("Cannot set timeout to 0, minimum value is 1")
```

Comment:
just a debug msg?
we are anyway returning the error can we remove this?

Reply:
error will only be returned only if I use `RunE`, but while using `RunE` it will print the entire help msg as well as mentioned https://github.com/redhat-developer/odo/pull/757#issuecomment-423912589

Reply:
Sorry not sure, I follow...
We are in a function inside the helper func to the cobra command right which can return anything(even an error) right?
How does RunE come into picture here?